dist/common/scripts/scylla_io_setup

```diff
@@ -178,6 +178,7 @@ def run_iotune():
                                        "--format", "envfile",
                                        "--options-file", etcdir() + "/scylla.d/io.conf",
                                        "--properties-file", etcdir() + "/scylla.d/io_properties.yaml",
+                                       "--duration=180",
```

Comment:
That's not very correct to just increase the duration, as iotune will still "include" the peak throughput measurement into result. The correct way to handle _that_ problem is https://github.com/scylladb/seastar/pull/2810

Reply:
@xemul I see you merged the warmup period bug, safe to drop the commit increasing the duration from here?